src/main/target/common.h

```diff
@@ -140,8 +140,8 @@
 
 #define USE_POWER_LIMITS
 
-#define NAV_FIXED_WING_LANDING
 #define USE_SAFE_HOME
+#define USE_FW_AUTOLAND
```

Comment:
Is this intended to be used for the changes in this PR at some point ?

Reply:
I had actually considered leaving Autoland out if necessary. Then I forgot to take it out.